src/Uno.UWP/WinRTFeatureConfiguration.cs

```diff
@@ -71,6 +71,11 @@ public static class FileTypes
 			/// Allows definition of custom file type MIME mappings.
 			/// </summary>
 			public static Dictionary<string, string> FileTypeToMimeMapping { get; } = new Dictionary<string, string>();
+
+			/// <summary>
+			/// Allows definition of custom file type UTType mappings.
```

Comment:
Missing docs article ?

Reply:
Will be part of the docs I am writing for pickers, not yet complete